examples/tfengine/iam_bindings.hcl

```diff
@@ -0,0 +1,73 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# {{$recipes := "../../templates/tfengine/recipes"}}
+
+data = {
```

Comment:
Are all of these necessary for the iam_bindings.hcl recipe?

Reply:
You're right, I will directly use the iam_bindings.hcl recipe instead of project.hcl and remove project specific information from here.